_config.yml

```diff
@@ -33,6 +33,21 @@ copyright_year: 2021
 description: Elemwave specialises in electromagnetic simulations
 logo: /assets/logo/elemwave.png
 
+title: Elemwave
```

Comment:
It seems that this was copied from the lines above:
https://github.com/elemwave/website/blob/ec70983bfb2ba5a9c2a2cb133e2d15dfcee39fe7/_config.yml#L23-L30

It should be removed